main.js

```diff
@@ -0,0 +1,200 @@
+const searchButton = document.getElementById("searchButton");
+const inputField = document.getElementById("inputField");
+const flag = document.getElementById("countryFlag");
+const displayCountryInfo = document.getElementById("countryInfo");
+const displayCountryName = document.getElementById("countryName");
+const displayCountryCurrency = document.getElementById("countryCurrency");
+const displayCountryLanguage = document.getElementById("countryLanguage");
+const errorMessage = document.getElementById("errorMessage");
+
+function clearInputField () {
+    inputField.value = null;
+}
+
+inputField.addEventListener("keyup", async (e) => {
+    try {
+        if (e.key === "Enter") {
+            const query = inputField.value;
+            response = await axios.get(
```

Comment:
Je zou hier de spread operator kunnen gebruiken, dan hoef je niet telkens `response.` ervoor te zetten als je een property van het object nodig hebt

Reply:
Zou je hiervan een voorbeeldje kunnen laten zien?

Reply:
Ja natuurlijk! Ik maak een voorbeeldje voor je zodat je het zelf kan toepassen:

```
const person = {
    firstName: "Piet",
    lastName: "Jansen",
};

// Manier zoals jij doet (wat niet verkeerd is)
const firstName = person.firstName;
const lastName = person.lastName;

// Met spread operator
const { firstName, lastName } = person;
```

Reply:
Aah op die manier.. ik googlede op spread operator en toen kwam ik alleen maar de {...} tegen.. en toen ik die probeerde werd het helemaal niks..
Helder!

Bedankt

Reply:
Ja die term was misschien ook niet juist kom ik achter 😛 Object destructering is daarentegen beter.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring